addon/components/o-s-s/currency-input.stories.js

```diff
@@ -0,0 +1,46 @@
+import hbs from 'htmlbars-inline-precompile';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Components/OSS::CurrencyInput',
+  component: 'currency-input',
+  argTypes: {
+    currency: {
+      description: 'The currency applied to the component (EUR, USD, etc.)',
+      table: {
+        type: {
+          summary: 'string'
+        },
+        defaultValue: { summary: false }
+      },
+      control: { type: 'text' }
+    },
+    value: {
+      description: 'The value applied to the input',
+      table: {
+        type: {
+          summary: 'number'
+        },
+        defaultValue: { summary: false }
+      },
+      control: { type: 'text' }
+    },
+    onChange: {
+      description: 'A callback that sends the modifications of the value & the currency back to the parent component'
+    }
```

Comment:
```suggestion
    type: { required: true },
    onChange: {
      description: 'A callback that sends the modifications of the value & the currency back to the parent component'
    }
```
To set "*" directly in the doc